File: guanabara-java/src/com/danillo/algoritmos/lista02/Exercicio23.java
package com.danillo.algoritmos.lista02;

import java.util.Scanner;

public class Exercicio23 {
    public static void main(String[] args) {
        Scanner entrada = new Scanner(System.in);

        System.out.println("Insira o seu nome: ");
        String nome = entrada.nextLine();
        System.out.println("Insira o seu gênero (M ou F): ");
        char genero = entrada.next().charAt(0);
        System.out.println("Insira o valor da compra: ");
        float valor = entrada.nextFloat();

        if(genero == 'F'){
            float desconto = valor * 13/100;
            float novoValor = valor - desconto;
            System.out.println("Valor com desconto (-13%): R$ " +novoValor);
        } else {
            float desconto = valor * 5/100;
            float novoValor = valor - desconto;
            System.out.println("Valor com desconto (-5%): R$ " +novoValor);
        }
    }
}

File: guanabara-python/src/lista02/exercicio23.py
nome = input("Insira o nome: ")
genero = input("Insira o gênero (M ou F): ")
valor_compra = float(input("Insira o valor da compra: "))

if genero == "M":
    desconto = (valor_compra * 5) / 100
    novo_valor_compra = valor_compra - desconto
    print(f"Valor: {valor_compra}. Valor (com 5% de desconto): {novo_valor_compra}")
else:
    desconto = (valor_compra * 13) / 100
    novo_valor_compra = valor_compra - desconto
    print(f"Valor: {valor_compra}. Valor (com 13% de desconto): {novo_valor_compra}")
